sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_patch.py

```diff
@@ -295,9 +295,7 @@ def begin_translation(
         """
 
     @overload
```

Comment:
Given we renamed the internal method _begin_start_translation --> _begin_translation, we'll need to update where we call this method in the _patch on L471 and in the async file

Reply:
Updated